tests/integ/scala/test_file_operation_suite.py

```diff
@@ -137,44 +149,67 @@ def test_put_with_one_file(session, temp_stage, path1, path2, path3):
     second_result = session.file.put(
         f"file://{path2}", stage_with_prefix, auto_compress=False
     )[0]
-    assert second_result.source == os.path.basename(path2)
-    assert second_result.target == os.path.basename(path2)
+    file_name_2 = os.path.basename(path2)
+    assert second_result.source == file_name_2
+    assert second_result.target == file_name_2
     assert second_result.source_size in (10, 11)
```

Comment:
Curious: why are we asserting it's in a range here? 🤔

Reply:
it's not a range, it's a set -- the source size can either be 10 or 11, and I remember this is because of different cloud storage implementation -- the source size on gcp is 10, while on azure and aws is 11.